BUILD.md

```diff
@@ -1017,6 +1018,32 @@ Android NNAPI Execution Provider can be built using building commands in [Androi
 
 ---
 
+### iOS
+
+#### Prerequisites
+* A Mac computer with latest macOS
+* Xcode
+   * Install Xcode from https://developer.apple.com/xcode/
+* General Info:
+   * iOS Architectures
+
+      | ios_sysroot     | arm64       | x86_64      |
+      |-----------------|:-----------:|:-----------:|
+      | iphoneos        | YES         | NO          |
+      | iphonesimulator | NO          | YES         |
+      armv7, armv7s and i386 architectures are not currently supported.
```

Comment:
Not clear to me how to interpret this but possibly as I know nothing about this setup. 

Is it that all mac's are x86_64 so you have to use the simulator if testing on a laptop/desktop and specify x86_64, and all iphones are arm64 so you have to build with arm64 to run on an actual device?